src/test/java/util/DynamodbTestUtils.java

```diff
@@ -0,0 +1,179 @@
+			}
+		}
+
+		throw new Exception("no network found");
+	}
+
+	private static String getLocalUrlForLocalDynamodb(final GenericContainer localDynamo) {
+		return "http://127.0.0.1:" + localDynamo.getFirstMappedPort();
+	}
+
+	/*
+	 * Create DynamodbTestUtils for aws connection
+	 */
+	public DynamodbTestUtils(final String user, final String pass) {
+		this("aws", "aws", user, pass);
+	}
+
+	private DynamodbTestUtils(final String localUrl, final String dynamoUrl, final String user, final String pass) {
+		this.localUrl = localUrl;
+		this.dynamoUrl = dynamoUrl;
+		this.dynamoUser = user;
+		this.dynamoPass = pass;
+		final StaticCredentialsProvider credentialsProvider = StaticCredentialsProvider
+				.create(AwsBasicCredentials.create(user, pass));
+		final DynamoDbClientBuilder clientBuilder = DynamoDbClient.builder().region(Region.EU_CENTRAL_1)
+				.credentialsProvider(credentialsProvider);
+		if (!this.dynamoUrl.equals("aws")) {
+			clientBuilder.endpointOverride(URI.create(this.dynamoUrl));
+		}
+		this.dynamoClient = clientBuilder.build();
+	}
+
+	public void pushItem() {
+		final HashMap<String, AttributeValue> itemValues = new HashMap<>();
+		itemValues.put("isbn", AttributeValue.builder().s("12398439493").build());
+		itemValues.put("name", AttributeValue.builder().s("funny Book").build());
+		final PutItemRequest request = PutItemRequest.builder().tableName("JB_Books").item(itemValues).build();
+		this.dynamoClient.putItem(request);
+	}
+
+	public int scan(final String tableName) {
+		final ScanResponse res = this.dynamoClient.scan(ScanRequest.builder().tableName(tableName).build());
+		LOGGER.trace(res.toString());
+		return res.count();
+	}
+
+	public void createTable(final String tableName, final String keyName) {
+		final CreateTableRequest request = CreateTableRequest.builder().tableName(tableName)
+				.attributeDefinitions(AttributeDefinition.builder().attributeName(keyName)
+						.attributeType(ScalarAttributeType.S).build())
+				.keySchema(KeySchemaElement.builder().keyType(KeyType.HASH).attributeName(keyName).build())
+				.provisionedThroughput(
+						ProvisionedThroughput.builder().readCapacityUnits(1L).writeCapacityUnits(1L).build())
+				.build();
+		this.dynamoClient.createTable(request);
+		this.tableNames.add(tableName);
+	}
+
+
+	public void deleteTable(final String tableName){
+		final DeleteTableRequest deleteRequest = DeleteTableRequest.builder().tableName(tableName).build();
+		this.dynamoClient.deleteTable(deleteRequest);
+		this.tableNames.removeIf(n -> n.equals(tableName));
+	}
+
+	public void deleteCreatedTables(){
+		for(final String tableName : this.tableNames){
+			this.deleteTable(tableName);
+		}
+	}
+
+	public void importData(final File asset) throws IOException, InterruptedException {
+		final Runtime runtime = Runtime.getRuntime();
+		String importCommand = "aws dynamodb batch-write-item --request-items file://" + asset.getPath();
+		if (!this.localUrl.equals("aws")) {
```

Comment:
this is not possible, as I use "aws" as address in the Create Connection. The reason therefore is, that exasol requires an address.

Reply:
In that case use the address part of the connection for something useful. For example the region in case of remote connection, IP/host and port in case of a "local" installation.

Reply:
I've already done so. ;-)